### PR TITLE
Copying an activity redirects to the iframed editor [#99311988]

### PR DESCRIPTION
**Pull request by dougmartin**
Note: there is a parallel rigse PR to support this feature (https://github.com/concord-consortium/rigse/pull/114)

### CODE REVIEW

File: app/controllers/lightweight_activities_controller.rb [hunk 1]
```diff
@@ -169,12 +169,25 @@ def duplicate
...
+        begin
+          json_response = JSON.parse(response.body)
+        rescue
+        end
+      end
+      if params['redirect_on_success']
+        if params['redirect_on_success'].include?(':activity_id') && json_response && json_response.has_key?("activity_id")
+          redirect_to params['redirect_on_success'].sub!(':activity_id', json_response['activity_id'].to_s)
+        else
+          redirect_to params['redirect_on_success']
```
Comment (knowuh) [inline app/controllers/lightweight_activities_controller.rb]: Seems like we could redirect to the portal, even if the publication wasn't successful.  How do we find out if we failed?

**Comment by dougmartin** [comment]
Good point.  I simplified the code since we know that the redirect url will always have the :activity_id template variable.  So on a publish failure then the LARA edit page is shown which shows the publishing failure - there is no way to iframe the failed publish so it is the best we can do.

**Comment by dougmartin** [comment]
@knowuh Do you have any other issues with this PR?

**Comment by knowuh** [comment]
Looks good.

**Pull request merged**